content/app/commands/goto-line-command.js

```diff
@@ -63,8 +64,12 @@ define("content/app/commands/goto-line-command", function(require, exports, modu
 		 */
 		_onSubmit : function (event) {
 			this._closePopup();
-			alert('Submitted value: ' + event.data);
-			this._triggerComplete();
+			var lineNumber = parseInt(event.data);
+			if (!isNaN(lineNumber)) {
```

Comment:
There is not need for !, since you have else branch. It should be:
if (isNaN(lineNumber)) ...
else ...